harness/determined/pytorch/lightning/_adapter.py

```diff
@@ -178,9 +180,10 @@ def __init__(self, context: PyTorchTrialContext) -> None:
 
         context.wrap_model(lightning_module)
 
-        pls = _LightningAdapterState(context, lightning_module, [])
+        pls = _LightningAdapterState(context, lightning_module, [], [])
         self._pls = pls
-        optimizers, _ = self.setup_optimizers_schedulers()
+        optimizers, lr_schedulers = self.setup_optimizers_schedulers()
```

Comment:
: D you can do something like this
```suggestion
        pls.optimizers, pls.lr_schedulers = self.setup_optimizers_schedulers()
```

Reply:
aah I didn't know, that's nicer.